### DISCUSSION TITLE
How to hide axis values in chart

### DISCUSSION THREAD

**Discussion by thushanr32** [PHPOffice/PhpSpreadsheet#3489] [Q&A]
How do you hide an axis's tick values. For example:

<img width="556" alt="Screenshot 2023-03-29 at 10 13 27 AM" src="https://user-images.githubusercontent.com/88255435/228566618-3309c606-492c-41c8-be46-78dca1432811.png">

**Comment by oleibman** [comment]
Does the following work for you?
```php
$axis->setAxisOptions('hidden', '1');
```

**Comment by thushanr32** [comment]
Yes, it does work. Thanks
